src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -725,7 +722,7 @@ trait ChoosesMode {
               seq :+ (vehicleOnTrip -> ParkingInquiry.init(
                 SpaceTime(geo.wgs2Utm(leg.beamLeg.travelPath.endPoint.loc), leg.beamLeg.endTime),
                 nextAct.getType,
-                VehicleManager.getReservedFor(veh.vehicleManagerId.get).get,
+                VehicleManager.getReservedFor(this.vehicleManagerId).get,
```

Comment:
What if we use a shared vehicle that has its own vehicle manager? In this case, we receive a wrong parking stall.